presto-benchto-benchmarks/src/main/resources/sql/presto/tpch/q22.sql

```diff
@@ -5,20 +5,20 @@ SELECT
 FROM 
   (
     SELECT 
-      substring(c.phone,1,2) AS cntrycode, 
+      substr(c.phone,1,2) AS cntrycode,
```

Comment:
Did you have any problem with `substring`? Both `substr` and `substring` are documented, with the latter being an alias for the former.